en/flight_controller/crazyflie2.md

```diff
@@ -105,24 +112,18 @@ with the crazyradio PA.
     pip install -r requirements.txt
     ```
 
-**Method 2:** 
-For systems that support [make](https://www.gnu.org/software/make/manual/html_node/Simple-Makefile.html), follow this alternative:
- * `pip install tox`
- * Navigate to the crazyflie-lib-python folder.
- * `make venv`
- * Activate the virtual environment: `source venv-cflib/bin/activate`
-
 To connect crazyflie with crazyradio, **launch cfbridge** by following these steps:
 - Switch on CF2 (which is already flashed with PX4 firmware) by pressing its ON button and wait for it to boot up.
 - Connect a Crazyradio PA via USB.
 - Navigate to the crazyflie-lib-python folder.
 - Activate the environment: `source venv-cflib/bin/activate` or `source venv/bin/activate` depending on the name of the venv folder you have.
 - `cd examples`
 - Launch cfbridge: `python cfbridge.py`
+> **Note** *Cfbridge* by default tries to initiate the radio link communication on channel 80 and with crazyflie address 0xE7E7E7E7E7. If you are using [multiple crazyflies and/or crazyradios](https://github.com/dennisss/cfbridge/blob/master/README.md#advanced-swarming) in the same room and want to use a different channel and/or address for each, first connect the crazyflie with QGC via a USB cable and change the syslink parameters (channel, address) in QGC. Next, launch the cfbridge by giving the same channel and address as the first and second arguments respectively, e.g: `python cfbridge.py 90 0x0202020202`
```

Comment:
I assume this bullet is part of the previous bullet point. Can you please indent 2 spaces?